pkg/traceability/traceability.go

```diff
@@ -241,10 +246,47 @@ func registerHealthCheckers(config *Config) error {
 		hcJob := &condorHealthCheckJob{
 			agentHealthChecker: ta,
 		}
+
+		// TBD. Remove in future when Jobs interface is complete
+		ta.hcJob = hcJob
+
 		_, err := jobs.RegisterIntervalJob(hcJob, config.Timeout)
 		if err != nil {
 			return err
 		}
+
+		// TBD. Remove in future when Jobs interface is complete
+		err = registerOldHealthChecker(hcJob, ta.host)
+		if err != nil {
+			return err
+		}
 	}
 	return nil
 }
+
+// TODO: From here down all temporary until Jobs interface finishes full implementation
+func registerOldHealthChecker(hcJob *condorHealthCheckJob, host string) error {
```

Comment:
Just being picky here, but can we just name it registerHealthChecker?    the comments are fine and indicate it will be removed in the future.  And your PR general comment is good.  So no need to call it "old".

Reply:
OK.